src/Database/Model.php

```diff
@@ -672,6 +672,17 @@ protected function performDeleteOnRelations()
     //
     // Adders
     //
+    
+    /**
+     * Add attribute casts for the model.
+     * 
+     * @param  array              $attributes
```

Comment:
What's with all the space between array and attributes?

Reply:
Heh, couldn't tell you... I guess that's just what my editor decided to do. I'll go ahead and trim that.